client/src/main/java/io/split/client/JsonLocalhostSplitChangeFetcher.java

```diff
@@ -29,10 +32,14 @@ public JsonLocalhostSplitChangeFetcher(InputStreamProvider inputStreamProvider)
     }
 
     @Override
-    public SplitChange fetch(long since, FetchOptions options) {
+    public SplitChange fetch(long since, long sinceRBS, FetchOptions options) {
         try {
             JsonReader jsonReader = new JsonReader(new BufferedReader(new InputStreamReader(_inputStreamProvider.get(), StandardCharsets.UTF_8)));
             SplitChange splitChange = Json.fromJson(jsonReader, SplitChange.class);
+            splitChange.ruleBasedSegments = new ChangeDto<>();
+            splitChange.ruleBasedSegments.d = new ArrayList<>();
+            splitChange.ruleBasedSegments.t = -1;
+            splitChange.ruleBasedSegments.s = -1;
```

Comment:
why are you setting this properties with a default value and reading from the file ?

Reply:
Sure